src/FileReader.java

```diff
@@ -0,0 +1,34 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+class FileReader {
+
+    /* Checking if the file is either txt or json  */
+
+    public static boolean isFileExists() {
```

Comment:
Name is not correct, you are not checking if file exists, you are only checking for the file type. The name of the method should reflect that.